app/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -537,6 +537,7 @@ public Study updateStudy(Study study, boolean isAdminUpdate) {
             study.setReauthenticationEnabled(originalStudy.isReauthenticationEnabled());
             study.setAccountLimit(originalStudy.getAccountLimit());
             study.setStudyIdExcludedInExport(originalStudy.isStudyIdExcludedInExport());
+            study.setAutoVerificationPhoneSuppressed(originalStudy.isAutoVerificationPhoneSuppressed());
```

Comment:
Non-admins can set AutoVerificationEmailSuppressed. Should this be made consistent?

Reply:
Seems fine for this (and any other setting that adjusts workflow) to be a developer setting.